publish.gradle

```diff
@@ -39,19 +60,32 @@ afterEvaluate {
                     }
                     developers {
                         developer {
-                            name = 'The Android Open Source Project'
+                            id = 'Firework'
+                            name = 'Firework Android Team'
+                            email = 'team-android@fireworkhq.com'
                         }
                     }
+
                     scm {
-                        connection = 'scm:git:https://github.com/google/ExoPlayer.git'
-                        url = 'https://github.com/google/ExoPlayer'
+                        connection = 'scm:git:git@github.com:loopsocial/ExoPlayer.git'
+                        url = 'https://github.com/loopsocial/ExoPlayer'
                     }
                 }
             }
         }
     }
+
+    signing {
+        def signingKeyId = System.getenv("GPG_SIGNING_KEY_ID")
+        def signingSecretKey = System.getenv("GPG_SIGNING_SECRET_KEY")
+        def signingPassword = System.getenv("GPG_SIGNING_PASSWORD")
+        useInMemoryPgpKeys(signingKeyId, signingSecretKey, signingPassword)
+
+        sign publishing.publications.release
+    }
 }
-tasks.withType(PublishToMavenRepository) { it.dependsOn lint, test }
+
+// tasks.withType(PublishToMavenRepository) { it.dependsOn lint, test }
```

Comment:
ℹ️ I don't get why, but tests are not passing and this line blocks the publishing task.